onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -4,3 +4,42 @@
 - All functions should not have the script() decorator. This is because
     we want to delay the compilation of the function.
 """
+from beartype.vale import Is
+from typing_extensions import Annotated
+
+from onnxscript.onnx_opset import default_opset as op
+
+
+def atenop(name):
+    """A no-op decorator for torch.aten operators."""
+
+    del name
+
+    def decorator(func):
+        return func
+
+    return decorator
+
+
+# TODO: put this in nn
+@atenop("aten::relu6")
```

Comment:
supporting decorators like this is tricky. Even though currently onnxscript does not produce any error messages when applied to this, I think the safer thing to do is to report an error message since we don't know the effect of the decorator in general. I wonder what you intend to use this for (in case there's a requirement that onnxscript could support).

Reply:
I intend to use the decorator to register the function to a dictionary-type container. The decorator should be a no-op from onnxscript's perspective.